docs/LANGUAGE_TRANSLATION.md

```diff
@@ -0,0 +1,188 @@
+# Language Translation
+
+> **Note:** development on this project is moving quickly, so parts of this document may be out of date. Feel free to ask questions in the `#tc-aid-translations` channel in Slack.
+
+1. Overview
+   1. The translation spreadsheet
+   2. Glossary
+   3. Special links
+2. Maintaining translations
+   1. Translating existing terms
+   2. Creating a new term
+   3. Adding a new language
+3. Technical details
+   1. Adding a term in markup (`data-translation-id`)
+   2. Getting a translated term in js code (`get`)
+   3. Setting the current language (`language`)
+   4. Running translation substitution (`translate`)
+
+## Overview
+
+The site translations work by loading a spreadsheet of translated terms, and replacing the hard-coded English terms on the page with the translated ones. If a translated term is missing in the spreadsheet, the English term will be used. The terms that need to be translated are specified **in the code**, it will **not** automatically find words on the page. 
+
+A basic reference for what kind of access is needed for various actions in the translation process. 
+
+| Action | Edit Spreadsheet | Update code |
+| :-----------------| :-: | :-: |
+| Translating terms* | No | No |
+| Adding translated terms | Yes | No |
+| Adding new terms | Yes | Yes | 
+| Adding a Language | Yes | Yes | 
+| Enabling a Language | No | Yes |
+
+*assuming the translator is working outside the main spreadsheet
+
+### The translation spreadsheet
+
+The spreadsheet used for defining translations is here: https://docs.google.com/spreadsheets/d/1m5QPNP6O8nsQsqJcQbwib_obpXRjmbYYGtwKzzg1l38/edit?pli=1#gid=0. You can request access in the slack channel `#tc-aid-translations`.
+
+Here's the basic format of the spreadsheet:
+
+| id | eng | spa | 
+| :--------- | :-------- |:--------- |
+| lang_name | English | Español |
+| lang_name_in_eng | English | Spanish |
+| language | Language | Idioma |
+| welcome | Welcome! | ¡Bienvenidos! |
+
+The first column (e.g. `lang_name`) and first row (e.g. `eng`) are **required** and must be filled out for any added row/column. The rest of the cells can be filled in as new translated terms are available.
```

Comment:
I think you mean "The first two rows (language key & lang_name) are required when adding a new column."